src/main/java/edu/cornell/kfs/pmw/batch/service/PaymentWorksVendorSupplierDiversityService.java

```diff
@@ -8,6 +8,6 @@
 
 public interface PaymentWorksVendorSupplierDiversityService {
     
-    List<VendorSupplierDiversity> buildSuppplierDivsersityListFromPaymentWorksVendor(PaymentWorksVendor pmwVendor);
+    List<VendorSupplierDiversity> buildSuppplierDiversityListFromPaymentWorksVendor(PaymentWorksVendor pmwVendor);
```

Comment:
This method still has a typo that needs fixing; there's an extra "p" that should be removed.

Reply:
ok